checkout-ui-custom/checkout6-custom.js

```diff
@@ -157,8 +157,14 @@
       allOptions.forEach(function (obj) {
         const currOption = obj.innerText.trim()
 
+        const isCreditCard = currOption.indexOf('Credit card') === 0
```

Comment:
``` suggestion
        const currOption = obj.innerText.trim().toLowerCase()

        const isCreditCard = currOption.indexOf('credit card') === 0
```
Avoiding some error validation in these cases: Credit Card, CREDIT CARD, and so on.

Reply:
Done, but this involved making a few other modifications